zeppelin-integration/pom.xml

```diff
@@ -94,21 +110,55 @@
       </exclusions>
       <scope>test</scope>
     </dependency>
-    
+
     <dependency>
-      <groupId>org.apache.commons</groupId>
-      <artifactId>commons-lang3</artifactId>
+      <groupId>org.apache.zeppelin</groupId>
+      <artifactId>spark-scala-2.11</artifactId>
```

Comment:
Depending on the spark-scala-2.11 or spark-scale-2.12 profile, one or the other dependency should be added.